frontend/src/app/modules/shared/songs-list/songs-list.component.ts

```diff
@@ -11,12 +13,26 @@ import { SongSortType } from '../../../models/song/song-sort-type';
 })
 export class SongsListComponent {
   @Input() songs: Song[];
+  @Input() isDraggable = false;
+  @Input() isEditable = false;
 
   @Output() clickMenuItem = new EventEmitter<{ menuItem: string, song: Song }>();
   @Output() clickDislike = new EventEmitter<number>();
 
   sortType: SongSortType | null = null;
 
+  constructor(private _songService: SongsService){ }
+
+  drop(event: CdkDragDrop<Song[]>) {
+    moveItemInArray(this.songs, event.previousIndex, event.currentIndex);
+
+    const orders = this.songs.map((s, index) => { return {id: s.id, order: index}});
```

Comment:
Change to 
const orders = this.songs.map((s, index) => ({id: s.id, order: index}));